ert_gui/simulation/run_dialog.py

```diff
@@ -191,6 +194,36 @@ def on_new_iteration(self, parent: QModelIndex, start: int, end: int) -> None:
 
             self._tab_widget.addTab(widget, f"Iteration {iter}")
 
+    @Slot(QModelIndex)
+    def _job_clicked(self, index):
+        node = index.internalPointer()
+        if node is None or node.type != NodeType.JOB:
+            return
```

Comment:
There are only two types of indices from this model, the invalid one (root index, should never be clickable) and valid ones.

```suggestion
        if not index.isValid():
            return
        node = index.internalPointer()
```